.sonarcloud.properties

```diff
@@ -0,0 +1,5 @@
+# Define the root directory(s) for main sources and tests
+sonar.sources=MAP,API/src, API/src
+# Exclude files in a test directory from the main source files
+sonar.exclusions=**/Reference/api_v1/**/*, API/Reference/api_v1/src/**/*
```

Comment:
these paths also overlap and might be causing the error, try taking one of them out

Reply:
done!